cesium/internal/testutil/fs.go

```diff
@@ -17,3 +19,61 @@ var FileSystems = map[string]FSFactory{
 		return MustSucceed(xfs.Default.Sub(dirName)), func() error { return xfs.Default.Remove(dirName) }
 	},
 }
+
+func ReplicateFS(srcFS, destFS xfs.FS) error {
```

Comment:
Do we need this function so we can test with memfs? If not why can't we just use a regular copy command? Also, name this function `CopyFS` instead of replicate.

Reply:
Yes, we need to be able to test with memfs

Reply:
Yes, we need to use it to test memfs